src/main/cli/settings.c

```diff
@@ -923,6 +924,9 @@ const clivalue_t valueTable[] = {
     { "yaw_deadband",               VAR_UINT8  | MASTER_VALUE, .config.minmaxUnsigned = { 0, 100 }, PG_RC_CONTROLS_CONFIG, offsetof(rcControlsConfig_t, yaw_deadband) },
     { "yaw_control_reversed",       VAR_INT8   | MASTER_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_OFF_ON }, PG_RC_CONTROLS_CONFIG, offsetof(rcControlsConfig_t, yaw_control_reversed) },
 
+// PG_PID_ADJUST    
+    { "pid_adjust_aux_channel",     VAR_UINT8  | MASTER_VALUE,  .config.minmax = { 0, 7 }, PG_PID_ADJUST, offsetof(pidAdjust_t, pid_adjust_aux_channel) },
```

Comment:
Two features mixed in one PR ... 

BTWE: isn't it possible to use existing adjustment mechanisms?

Reply:
@ledvinap: Correct. I am willing to let this slide under 'support for heli mixing', all encapsulated with the same conditional. But having it in two pull requests would be cleaner.

Reply:
@ledvinap No, when using different head speeds it’s unfortunately not. 
For example if flying at 2500 RPM, then switching to 3500 RPM mid flight at lest the P and D gains will have to be adjusted, otherwise it will crash.

Reply:
@mikeller I will remember and do better next time!

Reply:
Since a value of 0 means feature disabled, the range should be `.config.minmaxUnsigned = { 0, MAX_SUPPORTED_RC_CHANNEL_COUNT }` 

New files to review committed!